package.json

```diff
@@ -99,6 +99,11 @@
           "tags": [
             "telemetry"
           ]
+        },
+        "camel.languageSupport.JBangVersion": {
+          "type": "string",
+          "description": "Apache Camel JBang version used for JBang command shell executions.",
```

Comment:
@apupier may I ask you to give me some ideas what all we want to provide user in description here?.. my creative spirit is probably out of office already 😞😆

Reply:
- mention that it is for the command used internally by VS Code
- mention that it is affecting this extension only
- mention that each actions have different requirement of Camel JBang. Using a version close to the default version is recommended (not sure how to turn that nicely)

Reply:
![image](https://github.com/camel-tooling/camel-lsp-client-vscode/assets/18696866/2e919cca-e97f-4f30-b89c-64b472d5adb5)

WDYT?